packages/main/src/Button.ts

```diff
@@ -363,6 +389,9 @@ class Button extends UI5Element implements IFormElement {
 
 	async onBeforeRendering() {
 		const formSupport = getFeature<typeof FormSupport>("FormSupport");
+		if (this.type !== ButtonType.Button) {
```

Comment:
It think we need to add `&& !formSupport` - this.type !== ButtonType.Button && !formSupport
and only then log the warning, because the feature might be imported and the warning won't be needed

Reply:
thanks, I didn't notice I left it out. Done